routes/users.js

```diff
@@ -264,9 +264,9 @@ router.patch("/:userId", async (req, res) => {
 
 router.delete("/", async (req, res) => {
```

Comment:
nit: Very dangerous endpoint. Recommend commenting out until we have authorization implemented (if ever)

Reply:
true, will delete